src/%IPM/ResourceProcessor/ModuleExport.cls

```diff
@@ -0,0 +1,35 @@
+Class %IPM.ResourceProcessor.ModuleExport Extends %IPM.ResourceProcessor.Abstract
```

Comment:
This class is missing from our initial google doc so add it back to generate hsutil.xml. Also updated the google sheet